lib/cfg-grammar.y

```diff
@@ -931,6 +932,7 @@ stat_option
 	: KW_STATS_FREQ '(' nonnegative_integer ')'          { last_stats_options->log_freq = $3; }
 	| KW_STATS_LEVEL '(' nonnegative_integer ')'         { last_stats_options->level = $3; }
 	| KW_STATS_LIFETIME '(' positive_integer ')'      { last_stats_options->lifetime = $3; }
+  | KW_STATS_MAX_DYNAMIC '(' nonnegative_integer ')'   { last_stats_options->max_dynamic = $3; }
```

Comment:
What is the default value of this option? What will be on the documentation? As I see:

- `>0` means the limit of the dynamic stat clusters
- `0` means disable storing dynamic stat clusters
- `-1` means there is no limit - but it is not a valid value here...

Reply:
-1 is cannot be set by users in the config, it is the _default_ value, so when the option is not set then number of dynamic counters are unlimited as before.